src/uu/echo/src/echo.rs

```diff
@@ -59,6 +59,10 @@ fn parse_code(
     std::char::from_u32(ret)
 }
 
+// Clippy complatins that `.peek()` is never called on
+// `iter`. However, it is called inside the `parse_code()` function
+// that borrows `iter`.
+#[allow(clippy::unused_peekable)]
```

Comment:
we are getting this here:
```
ERROR: `cargo clippy`: unknown lint: `clippy::unused_peekable` (file:'src/uu/echo/src/echo.rs', line:65)
```

I guess we don't use the latest version in the CI

Reply:
I'll just remove this change for now.